manifests/init.pp

```diff
@@ -317,6 +318,7 @@
     # - $server_tokens
     # - $server_signature
     # - $trace_enable
+    # - $rewrite_lock
```

Comment:
we should [validate this variable](https://github.com/puppetlabs/puppetlabs-stdlib#validate_absolute_path),
```puppet
if $rewrite_lock {
  validate_absolute_path($rewrite_lock)
}
```